gulpfile.js

```diff
@@ -0,0 +1,17 @@
+var gulp = require('gulp');
+var less = require('gulp-less');
+var path = require('path');
+var minifyCSS = require('gulp-minify-css');
+var rename = require('gulp-rename');
+ 
+gulp.task('less', function () {
```

Comment:
does it make more sense for the task to be 'css' instead of 'less'? It's building css that's relevant, and I wouldn't expect the command to change if we happened to switch to SCSS or something else, for whatever reason.